### PR TITLE
update anon function to use arrow function

### PR DESCRIPTION
**Pull request by jared-esri**
@ldanzinger @anmacdonald Ready for review. Thanks!

Updating our samples to use the arrow function `()=>` where applicable.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_QMLSamples/Analysis/TraceUtilityNetwork/TraceUtilityNetwork.qml [hunk 1]
```diff
@@ -367,7 +367,7 @@ Rectangle {
                         deviceObjIds = [];
                         lineObjIds = [];
                         mapView.graphicsOverlays.get(0).graphics.clear();
-                        mapView.map.operationalLayers.forEach(function(layer){
+                        mapView.map.operationalLayers.forEach(layer => {
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Analysis/TraceUtilityNetwork/TraceUtilityNetwork.qml:370]: for these one-liners we can get rid of curly brace and really simplify `mapView.map.operationalLayers.forEach(layer => layer.clearSelection());`

Comment (ldanzinger) [reply ArcGISRuntimeSDKQt_QMLSamples/Analysis/TraceUtilityNetwork/TraceUtilityNetwork.qml:370]: there are other instances throughout where we can simplify a bit more

**Comment by jared-esri** [comment]
I reduced some code down to one line, https://github.com/Esri/arcgis-runtime-samples-qt/pull/1055/commits/89084d49ee7d941d2dc6737b69127e68392a8c47. I don't think this impacts readability but let me know if you think otherwise and I'll change it back. Otherwise I think the rest is good.

**Comment by ldanzinger** [comment]
@jared-esri this one is good to merge now that the codebase is open

**Pull request merged**